docs/tutorials/how-to-create-template.md

```diff
@@ -4,7 +4,7 @@ sidebar_position: 4
 
 # How to Create a Template
```

Comment:
What about rename the title to `Create/Write a Template`

Reply:
change to `Create your first template`